src/core.js

```diff
@@ -74,6 +74,24 @@ var QUnit,
 			}
 		}
 		return vals;
+	},
+	/**
+	 * Gets the keys of an object which are both enumerable and own
+	 *
+	 * @param {Object} obj
+	 * @return {Array} Array with the enumerable and own keys of the object given
+	 */
+	ownObjectKeys = function( obj ) {
```

Comment:
This is used only in `src/equiv.js`. Let's add it there. We want to reduce circular dependencies between QUnit modules, should add even more.